compiler/src/import/store/import.rs

```diff
@@ -37,21 +37,21 @@ impl<F: Field + PrimeField, G: GroupType<F>> ConstrainedProgram<F, G> {
             return Ok(());
         }
 
-        // // Fetch dependencies for the current import
-        // let imported_symbols = ImportedSymbols::new(import);
-        //
-        // for (name, symbol) in imported_symbols.symbols {
-        //     // Find imported program
-        //     let program = imported_programs
-        //         .get_import(&name)
-        //         .ok_or_else(|| ImportError::unknown_package(import.package.name.clone()))?;
-        //
-        //     // Parse imported program
-        //     self.store_definitions(program.clone(), imported_programs)?;
-        //
-        //     // Store the imported symbol
-        //     self.store_symbol(scope, &name, &symbol, program)?;
-        // }
+        // Fetch dependencies for the current import
+        let imported_symbols = ImportedSymbols::new(import);
+
+        for (name, symbol) in imported_symbols.symbols {
+            // Find imported program
+            let program = imported_programs
+                .get_import(&name)
+                .ok_or_else(|| ImportError::unknown_package(import.package.name.clone()))?;
+
+            // Parse imported program
+            self.store_definitions(program.clone(), imported_programs)?;
```

Comment:
Remove `.clone()` and change `store_definition` to pass by reference the `program`

Reply:
fixed by https://github.com/AleoHQ/leo/pull/434/commits/d3ab32822ed0c384c0984497d3b4f8c6594a4b67